docs/data-plane-configuration.md

```diff
@@ -0,0 +1,73 @@
+the data plane for its configuration. Additionally, the control plane updates the status of the resource (if it exists)
+to reflect whether it is valid or not.
+
+### Spec
+
+| name | description                                        | type              | required |
+| ---- | -------------------------------------------------- | ----------------- | -------- |
+| http | HTTP defines the NGINX HTTP context configuration. | [http](#spechttp) | no       |
+
+### Spec.HTTP
+
+| name      | description                                    | type                        | required |
+| --------- | ---------------------------------------------- | --------------------------- | -------- |
+| telemetry | Telemetry defines the telemetry configuration. | [telemetry](#spechttptelemetry) | no       |
+
+### Spec.HTTP.Telemetry
+
+| name    | description                                | type                    | required |
+| ------- | ------------------------------------------ | ----------------------- | -------- |
+| tracing | Tracing defines the tracing configuration. | [tracing](#spechttptelemetrytracing) | no       |
+
+### Spec.HTTP.Telemetry.Tracing
+
+| name       | description                                                                                                         | type   | required |
+| ---------- | ------------------------------------------------------------------------------------------------------------------- | ------ | -------- |
+| enable    | Enabled enables or disables OpenTelemetry tracing at the HTTP context. Default is false.                            | bool   | no       |
+| endpoint   | Endpoint specifies the address of OTLP/gRPC endpoint that will accept telemetry data.                               | bool   | yes      |
+| interval   | Interval specifies the tracing interval. Default is 5s.                                                             | string | no       |
+| batchSize  | BatchSize specifies the maximum number of spans to be sent in one batch per worker. Default is 512.                 | int    | no       |
+| batchCount | BatchCount specifies the number of pending batches per worker, spans exceeding the limit are dropped. Default is 4. | int    | no       |
+
+## Viewing and Updating the Configuration
+
+> For the following examples, the name `nginx-proxy-config` should be updated to the name of the resource that
+> was created by your installation.
+
+To view the current configuration:
+
+```shell
+kubectl -n nginx-gateway get nginxproxies nginx-proxy-config -o yaml
+```
+
+To update the configuration:
+
+```shell
+kubectl -n nginx-gateway edit nginxproxies nginx-proxy-config
+```
+
+This will open the configuration in your default editor. You can then update and save the configuration, which is
+applied automatically to the data plane.
+
+To view the status of the configuration:
+
+```shell
+kubectl -n nginx-gateway describe nginxproxies nginx-proxy-config
+```
```

Comment:
Per my other comment regarding whether or not we need an example, I would update this doc maybe with an example Jaeger deployment and telemetry config that would send traces there.